dask/base.py

```diff
@@ -811,6 +836,9 @@ def normalize_array(x):
             except (TypeError, UnicodeDecodeError):
                 try:
                     data = hash_buffer_hex(pickle.dumps(x, pickle.HIGHEST_PROTOCOL))
+                    warnings.warn(
+                        normalize_token_array_pickle_warning, NormalizeTokenWarning
+                    )
```

Comment:
This seems extreme.  This would flood users screens with warnings if they choose to include numpy arrays in their workloads.  This is a very common thing to do, and is sometimes the right thing to do.  We can't flood their screens in this case.

Reply:
@mrocklin thanks for the review!  This will only warn the user when they include data in a numpy array that needs to be pickled, not when they use any numpy array.  Also, it will not flood screens: see https://stackoverflow.com/questions/22661745/why-only-one-warning-in-a-loop